packages/core/src/__integrationtests__/Attestation.spec.ts

```diff
@@ -1,9 +1,3 @@
-/**
- * @packageDocumentation
- * @group integration/attestation
```

Comment:
We have to add the `@group` tags to the integration tests, so that they are triggered correctly via the `yarn run test:integration:run` command